includes/class-wc-payments-order-service.php

```diff
@@ -1747,7 +1785,7 @@ private function compose_dispute_url( $charge_id ) {
 		return add_query_arg(
 			[
 				'page' => 'wc-admin',
-				'path' => '/payments/transactions/details',
+				'path' => rawurlencode( '/payments/transactions/details' ),
```

Comment:
Not sure how this got reverted from https://github.com/Automattic/woocommerce-payments/blob/1137c625bd02502e003ebd8b09de473e82ea4b3a/includes/class-wc-payments-order-service.php#L1752 🤷